xml/ay_bigfile.xml

```diff
@@ -4156,6 +4156,11 @@ config:type="boolean"&gt;true&lt;/diag&gt;</screen>
       </tgroup>
    </informaltable>
 
+   <para>
+   If a primary language is unknown, it will be automatically changed into
+   the default one, which is <literal>en_US</literal>.
```

Comment:
I think in the current configuration of the sentence, the "it" is a bit unclear. Would the following work? "If the configured value for the primary language is unknown, it will be reset to the default, <literal>en_US</literal>."